constants.bal

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+//Base URL
+const string BASE_URL = "https://www.googleapis.com";
+
+// URL encoding
+const string ENCODING_CHARSET = "utf-8";
+
+//Symbols
+const string QUESTION_MARK = "?";
+const string PATH_SEPARATOR = "/";
+const string EMPTY_STRING = "";
+const string WHITE_SPACE = " ";
+const string FORWARD_SLASH = "/";
+const string DASH_WITH_WHITE_SPACES_SYMBOL = " - ";
+const string COLON = ":";
+const string EXCLAMATION_MARK = "!";
+const string EQUAL = "=";
+const string _ALL = "*";
+
+// Regex constants
+const string _FILE = "/file/";
+const string _FOLDER = "/folders/";
+const string _WORKSPACE_DOC = "/edit";
+const int INT_VALUE_8 = 8;
+const int INT_VALUE_39 = 39;
+const int INT_VALUE_41 = 41;
+const int INT_VALUE_42 = 42;
+const int INT_VALUE_83 = 83;
+
+// Drive
+const string DRIVE_URL = "https://www.googleapis.com";
+const string REFRESH_URL = "https://www.googleapis.com/oauth2/v3/token";
+const string DRIVE_PATH = "/drive/v3";
+const string ABOUT = "/about";
+const string UPLOAD = "/upload";
+const string UPLOAD_TYPE = "uploadType";
+const string TYPE_MEDIA = "media";
+const string TYPE_MULTIPART = "multipart";
+const string TYPE_RESUMABLE = "resumable";
+const string FILES = "/files";
+const string CHANGES = "/changes";
+const string COPY = "/copy";
+const string CHANNELS = "/channels";
+const string STOP = "/stop";
+const string Q = "q";
+const string MIME_TYPE = "mimeType";
+const string AMPERSAND = "&";
+const string PAGE_TOKEN = "pageToken";
+
+// Error
+const string ERR_FILE_RESPONSE =  "Error occurred while constructing DriveResponse record.";
+const string ERR_DRIVE_INFO_RESPONSE =  "Error occurred while constructing DriveInfo record.";
+const string ERR_JSON_TO_FILE_CONVERT =  "Error occurred while constructing File record.";
```

Comment:
Only one space should be added after the =

Reply:
Thank you @LakshanSS